gtest-parallel

```diff
@@ -424,4 +472,6 @@ if options.print_test_times:
               if times.get_test_time(test_binary, test) is not None)
   for (time_ms, test_binary, test) in ts:
     print "%8s %s" % ("%dms" % time_ms, test)
+if options.dump_json_test_results:
+  test_results.dump_to_file_and_close()
```

Comment:
Implement this in the IgnoreTestResults() class too and make it unconditional.

Also add an empty line before this one.